app/docs/async-api.yaml

```diff
@@ -0,0 +1,80 @@
+asyncapi: 2.4.0
+info:
+  title: FFC Payment Statement Constructor
+  version: '1.0.0'
+  description: Construct data for Payment Statements
+channels:
+  return/message:
+    subscribe:
+      message:
+        $ref: '#/components/messages/ReturnMessage'
+components:
+  messages:
+    ReturnMessage:
+      name: Return Message
+      contentType: application/json
+      payload:
+        $ref: '#/components/schemas/ReturnMessage'
+  schemas:
+    ReturnMessage:
+      type: object
+      required:
+      - sourceSystem
+      - invoiceNumber
+      - frn
+      - currency
+      - value
+      - settlementDate
+      - reference 
+      - settled
+      properties:
+        sourceSystem:
+          $ref: '#/components/schemas/SourceSystem'
+        invoiceNumber:
+          $ref: '#/components/schemas/InvoiceNumber'
+        frn:
+          $ref: '#/components/schemas/FRN'
+        currency:
+          $ref: '#/components/schemas/Currency'
+        value:
+          $ref:'#/components/schemas/Value'
```

Comment:
Interesting tht this 1 isn't being picked up correctly, try loading it in here and see if it renders properly: https://studio.asyncapi.com/

There are some rouge indentation differences (at least shown on GH) which might be problematic

Reply:
some spacing issues and a typo

Reply:
nice catch